<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditShell">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonFont">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="checkBoxConfirmClosing">
         <property name="text">
          <string>Ask for conformation when closing</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="MarginSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBoxLSpacing">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_16">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Font Unocode :</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="checkBoxBlinking">
         <property name="text">
          <string>Blinking Cursor</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Font:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBoxUnicode"/>
         </item>
         <item>
          <widget class="QFontComboBox" name="fontComboBox">
           <property name="writingSystem">
            <enum>QFontDatabase::Any</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Lines Spacing</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Cursor:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" rowspan="2">
        <widget class="QComboBox" name="comboBoxCursor">
         <item>
          <property name="text">
           <string>BlockCursor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UnderlineCursor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IBeamCursor</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Margin :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ScrollBar Position :</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>128</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxFont">
         <property name="text">
          <string>All fonts</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="ScrollBarComboBox">
         <item>
          <property name="text">
           <string>No ScrollBar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ScrollBar Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ScrollBar Right</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Shell:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="checkBoxNativeWindow">
         <property name="text">
          <string>Native Window</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>this setting requires a restart to take effect.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabThemes">
      <attribute name="title">
       <string>Themes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>OPacity:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxOPacity">
              <property name="suffix">
               <string> %</string>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Color Sheme :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="themesComboBox">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Default</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Get Scheme</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widgetCostumColor" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="hLayoutForColor">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Font Color:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="hLayoutBackColor">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>BackGround Color:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="hLayoutCursorColor">
               <item>
                <widget class="QCheckBox" name="checkBoxCursColor">
                 <property name="text">
                  <string>Cursor Color</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Scheme :</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayoutBtn">
               <item row="0" column="4">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Blue</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_25">
                 <property name="text">
                  <string>Black</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Yellow</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="6">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Cyan</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Green</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_23">
                 <property name="text">
                  <string>Red</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="7">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>White</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Magenta</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Keys</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>ADD Tab</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Copy</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Paste</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Zoom In</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Zoom Out</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Settings</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Quit</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Opacity Decrase</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Opacity Incrase</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string>9</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxOPacity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxOPacity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
